models/user/user.go

```diff
@@ -261,38 +259,38 @@ func (u *UserStruct) Projects() []*project.ProjectStruct {
 	return slice
 }
 
-func (u *UserStruct) Save() bool {
+func (u *UserStruct) Save() error {
```

Comment:
__[golint]__
* [golint] exported method UserStruct.Save should have comment or be unexported